### PR TITLE
Needs more beans

### PR DESCRIPTION
**Pull request by jminyardjc**
Added more beans

### CODE REVIEW

**Review by sam11385 — CHANGES_REQUESTED** [review]
Delete line

**Review by sam11385 — CHANGES_REQUESTED** [review]
remove template

**Comment by jminyardjc** [comment]
@mattbarrio  you get to decide. 😄

**Comment by ddouglassjc** [comment]
Most perfect PR I've ever seen.

File: templates/header.tmpl [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <main class="content" style="max-width: 1400px">
       <header class="container">
         <span>My name is <s>Sam</s> Matt. This site celebrates the best food in the world - beans! 
-        Click this <span id="magic">magic bean</span> for more AI generated BEANS things! »»»<button id="magic-beans" class="button padding-large white border"><img src="/static/img/logo.png"></button>«««</span>
+        Click this <span id="magic">magic bean</span> to experience the greatness of BEANS! »»»<button id="magic-beans" class="button padding-large white border" onclick="window.location.href='https://www.youtube.com/watch?v=wEEuzUGEWws';"><img src="/static/img/logo.png"></button>«««</span>
```
Comment (mattbarrio) [inline templates/header.tmpl]:
```suggestion
        Click this <span id="magic">magic bean</span> to experience the greatness of BEANS! »»»<button id="magic-beans" class="button padding-large white border" onclick="window.open('https://www.youtube.com/watch?v=wEEuzUGEWws','_blank')"><img src="/static/img/logo.png"></button>«««</span>
```

**Review by mattbarrio — COMMENTED** [review]
A slight adjustment on the script to open a new window. 

More tabs with more beans == winning

**Pull request merged**